config/rbac/instascale_role.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: instascale-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - machineset
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - machineset/status
+  verbs:
+  - get
```

Comment:
Are MachineSet APIs really in the `apps` group? It seems it's a wrong duplicate of the `machine.openshift.io` one.

Reply:
nope, I didn't think they were in the `apps` group, I thought first class objects like replicasets, deployments etc were.

but I was just copying what was in the [original role](https://github.com/project-codeflare/instascale/blob/2c7538eb8f93b77a768bd774bf963f37a0c41b23/config/rbac/role.yaml#L39-L56) to try get it working.

I can't really figure out why this is the case, even getting rid of those and allowing the wildcard on teh machine.openshift.io group doesn't returns the same error as ye got. 
I also can't seem to find a machinepool resource for any api group in any of the docs. 
Any ideas here?

Reply:
Actually, I don't think the error is related to the RBAC changes. I suspect there is an issue with the OCM client configuration.